pkg/kubelet/apis/podresources/v1alpha1/api.pb.go

```diff
@@ -17,36 +17,23 @@ limitations under the License.
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
 // source: api.proto
 
-/*
```

Comment:
Can you confirm that these updates were made by running `./hack/update-generated-pod-resources.sh`  I'm surprised it removed the comments at the top.